extensions/wikia/JWPlayerTag/JWPlayerTag.i18n.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$messages = [];
+
+$messages['en'] = [
+	'jwplayer-tag-desc' => 'JWPlayer Parser Tag for embedding video into articles',
+	'jwplayer-tag-could-not-render' => 'Failed to render the JWPlayer tag. Please check if all required parameters are in place.',
+];
+
+$messages['qqq'] = [
+	'jwplayer--tag-desc' => '{{desc}}',
+	'jwplayer-tag-could-not-render' => 'Error message shown when the JWPlayer tag validation failed.',
```

Comment:
maybe we should explicitly say that we need `media-id` here?

Reply:
Discussed offline -> We'll leave generic message.